static/app/utils/buttonClick/useButtonClick.tsx

```diff
@@ -0,0 +1,77 @@
+import {useCallback, useContext} from 'react';
+
+import {ButtonProps} from 'sentry/components/button';
+import HookStore from 'sentry/stores/hookStore';
+import {OrganizationContext} from 'sentry/views/organizationContext';
+import {RouteContext} from 'sentry/views/routeContext';
+
+type Props = ButtonProps;
+
+export const trackButtonClick = HookStore.get('analytics:track-button-clicks')[0];
```

Comment:
@roggenkemper yea I don't think this will work because the bookstore might not be initialized yet. In either case, I don't think we really need to export it like this.